app/lib/actions/pulp3/orchestration/content_view_version/export.rb

```diff
@@ -50,56 +50,32 @@ def plan(content_view_version:, destination_server: nil,
                                      chunk_size: chunk_size,
                                      from_content_view_version_id: from_content_view_version&.id)
 
-              plan_self(exporter_data: action_output[:exporter_data], smart_proxy_id: smart_proxy.id,
-                        destination_server: destination_server,
-                        content_view_version_id: content_view_version.id,
-                        from_content_view_version_id: from_content_view_version&.id)
-
+              history_output = plan_action(
+                ::Actions::Katello::ContentViewVersion::CreateExportHistory,
+                smart_proxy_id: smart_proxy.id,
+                exporter_data: action_output[:exporter_data],
+                pulp_href: action_output[:exporter_data][:pulp_href],
+                content_view_version_id: content_view_version.id,
+                from_content_view_version_id: from_content_view_version&.id,
+                destination_server: destination_server
+              ).output
               plan_action(::Actions::Pulp3::ContentViewVersion::DestroyExporter,
-                            smart_proxy_id: smart_proxy.id,
-                            exporter_data: action_output[:exporter_data])
+                smart_proxy_id: smart_proxy.id,
+                exporter_data: action_output[:exporter_data])
+              plan_self(exporter_data: action_output[:exporter_data], smart_proxy_id: smart_proxy.id,
```

Comment:
`plan_self` should normally be at the bottom of `plan` so I moved it

Reply:
I 'd recommend against this. You 'd want plan_self to run here before `::Actions::Pulp3::ContentViewVersion::DestroyExporter`

Reply:
Within `plan` the `input` and `output` values will be unpopulated, like in [the above comment](https://github.com/Katello/katello/pull/9211#discussion_r591857535), but in `run` they will be the actual values.  Due to this I think it should still work okay?

I checked and `exporter_data` does show up with the correct values in the UI (Raw tab):
```json
  "exporter_data": {
    "pulp_href": "/pulp/api/v3/exporters/core/pulp/b1eba19c-eaa9-4625-b2eb-342858321a6f/",
    "pulp_created": "2021-03-10T19:51:46.682+00:00",
    "name": "Test_Organization_Test_CV_1.0",
    "path": "/var/lib/pulp/exports/Test_Organization/Test_CV/1.0//2021-03-10T19-51-46-00-00",
    "repositories": [
      "/pulp/api/v3/repositories/rpm/rpm/8af5dca5-0d1c-4dbc-9a18-7b4df2766de1/"
    ]
  },
```

(However, I did run into a similar issue when I tried to plan the `CreateExportHistory` action after the `DestroyExporter` action.  That was because the pulp API request for the exporter returned a 404 after it was destroyed.)

Reply:
Oh I see you moved that to an action before. LGTM